.eslintrc.json

```diff
@@ -52,11 +52,11 @@
         "message": "Please replace `any` with a specific type"
       },
       {
-        "selector": "CallExpression[callee.object.name=/^(EditorState|NodeSelection|TextSelection|DecorationSet|ProsemirrorNode|MarkType)$/][callee.property.name=create]:not([typeParameters])",
+        "selector": "CallExpression[callee.object.name=/^(EditorState|NodeSelection|TextSelection|ProsemirrorNode|MarkType)$/][callee.property.name=create]:not([typeParameters])",
         "message": "Please provide a generic to avoid implicit `any`"
       },
       {
-        "selector": "CallExpression[callee.object.name=/^(EditorState|NodeSelection|TextSelection|DecorationSet|ProsemirrorNode|MarkType)$/][callee.property.name=create][typeParameters.params.0.type=TSAnyKeyword]",
+        "selector": "CallExpression[callee.object.name=/^(EditorState|NodeSelection|TextSelection|ProsemirrorNode|MarkType)$/][callee.property.name=create][typeParameters.params.0.type=TSAnyKeyword]",
```

Comment:
I think a better approach to all of this now that we have the ability to patch types is to remove the need to do generics at all. We can't copy this approach for `TextSelection` as you can get to the root schema from that type, which would then be typed as any, unlike with `DecorationSet`. This is fine for now, just saying it's not ideal in the long run.